.gitattributes

```diff
@@ -0,0 +1,10 @@
+* text=auto
+
+.gitattributes export-ignore
+.gitignore export-ignore
+composer.lock export-ignore
```

Comment:
This is already excluded by `.gitignore`, so it shouldn't exist in the repository in the first place.
```suggestion
```